src/core/installAddon.js

```diff
@@ -222,6 +222,7 @@ type WithInstallHelpersProps = {|
   addon: AddonType | null,
   defaultInstallSource: string,
   location: ReactRouterLocationType,
+  version: AddonVersionType | null,
```

Comment:
Technically I think this is `version?:` because not all wrapped components will define it.